docs/api/browser-window.md

```diff
@@ -662,6 +662,13 @@ Emitted when details regarding a requested resource is available.
 
 Emitted when a redirect was received while requesting a resource.
 
+### Event: 'page-favicon-set'
+
+* `event` Event
+* `favicons` [String]
+
+Emitted when page receives favicon urls.
```

Comment:
"urls" as far as I know, a page only has one favicon, which means only one wrong.
Was it introduced recently?